### DISCUSSION TITLE
希望多P视频可以自动跳转到上次观看的集数

### DISCUSSION THREAD

**Discussion by fate1ess** [the1812/Bilibili-Evolved#1935] [功能建议 / Ideas]
每次从收藏，或者稍后再看打开多P视频，都是从第一P开始。
希望可以记忆上次观看的P，自动从上次的P开始播放。

**Comment by Geeyun-JY3** [comment]
以前有个 `自动从历史记录点播放` 功能，后来 B 站加了个类似的功能后 [暂时关掉](https://github.com/the1812/Bilibili-Evolved/releases/tag/v1.11.5) 了这个功能。

你要的多 p 其实 B 站有记忆的，不过目前还不能自动，需要你手动点~~确实需要手快~~
![image](https://user-images.githubusercontent.com/49396421/121622749-1709dd80-caa1-11eb-9d01-c8e48f5e1891.png)

相关 issue：#1071、#1191
